CHANGELOG.md

```diff
@@ -11,13 +11,18 @@ the code was deployed.
 
 ## [Unreleased]
 
+### Added
+
+- Added 'View Client' button to Internal Dashboard-Location page (CU-8678vn1r5).
```

Comment:
I would suggest adding the clickup link for the task in here

Reply:
Thanks! Ahmad, as per our conversation we will keep the clickup link out going forward.